src/main/resources/data/minecolonies/loot_tables/recipes/sifter_gravel.json

```diff
@@ -123,7 +123,7 @@
       {
         "name": "Gravel_Diamond_Mesh",
         "conditions": [
-            {"condition": "minecraft:entity_properties", "entity": "this", "predicate" : { "equipment": { "mainhand": {"item" : "minecraft:diamond"} } }}
+            {"condition": "minecraft:entity_properties", "entity": "this", "predicate" : { "equipment": { "mainhand": {"item" : "minecolonies:siftrer_mesh_diamond"} } }}
```

Comment:
```suggestion
            {"condition": "minecraft:entity_properties", "entity": "this", "predicate" : { "equipment": { "mainhand": {"item" : "minecolonies:sifter_mesh_diamond"} } }}
```

Reply:
You should be able to use a `match_tool` condition on these anyway, it's shorter.  Or adopt the separate loot tables from my JEI branch to eliminate the condition entirely.